### PR TITLE
Clarify certificate rotation process


### PR DESCRIPTION
**Pull request by evrardjp**
The explanation of the rotation of the certificates was a little bit terse.
This should clarify how rotation happens.

Closes: https://github.com/SUSE/doc-caasp/issues/714
Closes: https://github.com/SUSE/doc-caasp/issues/715

### CODE REVIEW

File: adoc/architecture-description.adoc [hunk 1]
```diff
@@ -380,6 +380,9 @@ Certificates are stored under `/etc/kubernetes/pki` on the control plane nodes.
 |etcd/peer.key                |RSA      |2048
 |===
 
+Please refer to the _{productname} Admin Guide_ for more information on
```
Comment (r0ckarong) [inline adoc/architecture-description.adoc]: You can replace this with: 

```
Please refer to the link:{docurl}single-html/caasp-admin/#_certificates[{productname} Administration Guide] for more information on the rotation of certificates.
```

This automatically rewrites the `{docurl}` attribute to the correct location for the document and links to the certs section.

File: adoc/architecture-description.adoc [hunk 2]
```diff
@@ -434,6 +437,9 @@ Certificates are stored under `/var/lib/kubelet/pki` on the worker nodes.
 |kubelet.key |RSA      |2048
 |===
 
+Please refer to the _{productname} Admin Guide_ for more information on
```
Comment (r0ckarong) [inline adoc/architecture-description.adoc]: Same as above. Replace with:

```
Please refer to the link:{docurl}single-html/caasp-admin/#_certificates[{productname} Administration Guide] for more information on the rotation of certificates.
```

File: adoc/architecture-description.adoc (hunk 1 above)
Comment (evrardjp) [reply adoc/architecture-description.adoc]: I took this from the another guide. I am fine with changing it, but shouldn't it be changed everywhere else, in a different PR?

Comment (r0ckarong) [reply adoc/architecture-description.adoc]: Either way is fine. You can just fix this PR and tell me where you found the other place and I'll fix it there. If you already know and it's only one or two places feel free to add commits fixing this to this PR here.

**Comment by evrardjp** [comment]
@r0ckarong updated.

**Comment by evrardjp** [comment]
What's holding this up? Can we merge it? Do you need technical reviewers?

**Comment by r0ckarong** [comment]
> What's holding this up? Can we merge it? Do you need technical reviewers?

Simply too busy with other things. I will do a final review and build validation later today and merge it. I've got it assigned to me and in this milestone so it will be done when I get around to it.
This is not a high pressure P1/L3 scenario. Please be patient :)

**Pull request merged**